css/style.css

```diff
@@ -9,3 +9,89 @@ header {
   background: url("../images/header_bg.jpg") no-repeat center top;
   background-size: cover;
 }
+
+body {
+	font-family: open sans, sans-serif;
+}
+
+#title {
+	color: white;
+	font-size: 20px;
+	font-weight: 800;
+	text-align: left;
+	margin-left: 300px;
+	padding: 20px;
+}
+
+h2 {
+	color: white;
+	font-size: 40px;
+	font-weight: 400;
+	text-align: center;
+	padding: 70px auto;
+}
+
+#getitnow {
+	color: black;
+	font-weight: bold;
+	text-align: center;
+	background-color: #f9e42e;
+	margin: 20px 520px;
```

Comment:
Switch this margin to `margin: 20px auto;` then add `width: 180px;`. This will float the button in the center of the screen.